.github/workflows/package.yml

```
@@ -333,12 +333,13 @@ jobs:
 
       - name: Docker Hub Description
         uses: peter-evans/dockerhub-description@v3
+        if: ${{ github.event_name == 'release' }} && env.IS_DEPLOYABLE == 'true'
```

```suggestion
        if: github.event_name == 'release' && env.IS_DEPLOYABLE == 'true'
```
> `${{ <expression> }}`
> When you use expressions in an `if` conditional, you may omit the expression syntax (`${{ }}`) because GitHub automatically evaluates the `if` conditional as an expression. For more information about `if` conditionals, see "[Workflow syntax for GitHub Actions](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif)."

from: https://docs.github.com/en/actions/learn-github-actions/expressions

---

thanks!